css/style.css

```diff
@@ -464,10 +496,270 @@ div .taggs {
   color: #fff;
 }
 
-.shape {
-  margin-left: 30%;
-  background: #0d151e;
-  border-radius: 10px;
-  width: 134px;
-  height: 5px;
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    background: #e5e5e5;
+  }
+
+  #main-page {
+    left: 0;
+    height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: #fff;
+    background-image: url(/images/desktop_main/Header.svg);
+    background-size: 97%;
+    background-repeat: no-repeat;
+    background-position: 20px 80px;
+    margin-bottom: 4rem;
+    padding: 10px 24px;
+  }
+
+  #navbar {
+    overflow: hidden;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    margin-left: 8px;
+  }
+
+  .my-logo {
+    margin: 16px 0 0 100px;
+  }
+
+  .navbar-menu {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 1rem;
+    margin-right: 10%;
+  }
+
+  .navbar-menu a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .headline {
+    padding: 350px 260px 255px;
+    margin-top: 20px;
+  }
+
+  /* ////// Projects /////// */
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem 0;
+  }
+
+  .work-page-left {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-content: center;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .work-page-right {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .card1-content,
+  .card3-content {
+    margin: 30px 20px 0 0;
+  }
```

Comment:
- [ ] At the width of `900px`, the main title and info row get collapsed together. I will suggest adding some margin between them to make it pixel perfect to the figma design.
your design
![jh](https://user-images.githubusercontent.com/96683843/213528263-57065573-9f00-435a-a416-fd0017942dee.jpg)